src/SprykerEco/Yves/Heidelpay/Form/DataProvider/InvoiceSecuredB2cDataProvider.php

```diff
@@ -26,11 +26,11 @@ public function getData(AbstractTransfer $quoteTransfer): QuoteTransfer
             $quoteTransfer->setPayment(new PaymentTransfer());
         }
 
-        if ($quoteTransfer->getPayment()->getHeidelpayInvoiceSecuredB2C() !== null) {
+        if ($quoteTransfer->getPayment()->getHeidelpayInvoiceSecuredB2c() !== null) {
```

Comment:
Why it was done in the first place? it changes a lot, but there is no real value in this change